de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/core/ui/util/CollaborationUtils.java

```diff
@@ -301,8 +308,9 @@ private static String getShareProjectDescription(
      * @param sarosSession
      * @return
      */
-    private static Map<IProject, List<IResource>> acquireResources(
-        List<IResource> selectedResources, ISarosSession sarosSession) {
+    private static Map<IReferencePoint, List<IResource>> acquireResources(
+        List<IResource> selectedResources, ISarosSession sarosSession,
+        IReferencePointManager referencePointManager) {
 
         Map<IProject, Set<IResource>> projectsResources = new HashMap<IProject, Set<IResource>>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the type specification in this constructor call with the diamond operator ("<>"). ([squid:S2293](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2293))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJq4IkEeD8QRkTSvJ&open=AWcmJq4IkEeD8QRkTSvJ)

[ISSUE_KEY]: AWcmJq4IkEeD8QRkTSvJ